try
{ 
	// Código que pode lançar uma exceção 
	int result = 10 / 0;
 } 
catch (tipo_do_erro e)
{ 
	// Tratamento da exceção System.out.println("Erro: Divisão por zero!");
	throw e; // Relança a exceção 
}